source/user-manual/api/rbac/configuration.rst

```diff
@@ -6,22 +6,25 @@ Configuration
 =============
 Before modifying RBAC policies, roles and more, it is necessary to log into the API. You can find a detailed guide on how to do it within the :ref:`Getting started <api_getting_started>` section of the API.
 
-The endpoints required to manage RBAC belong to the *Security* group. To run them, this configuration guide uses the API interface. However, the command-line ``cURL`` can be used instead as explained in :ref:`Getting started <api_getting_started>`.
-
 Set RBAC mode
 -------------
-You can modify the RBAC mode (which we mentioned in the :ref:`How it works <api_rbac_how_it_works>` section). This is a security configuration so it is necessary to use the security endpoint ``/security/config``, specifically the ``PUT`` method. Just write the following setting inside the request body of the endpoint, setting the desired mode (white/black).
+You can modify the RBAC mode (which we mentioned in the :ref:`How it works <api_rbac_how_it_works>` section). This is a security configuration so it is necessary to use the security endpoint ``/security/config``, specifically the ``PUT`` method. To do so, you can run the following cURL command, replacing your token and the desired mode (white/black).
```

Comment:
If the user is reading the getting_started it is possible this is the first document he is reading. Please avoid phrases like "we mentioned in", the user might have not read it.